app/src/main/java/com/lightricks/feedexercise/ui/feed/FeedViewModel.kt

```diff
@@ -23,8 +63,7 @@ open class FeedViewModel : ViewModel() {
     }
 
     fun getFeedItems(): LiveData<List<FeedItem>> {
-        //todo: fix the implementation
-        return MutableLiveData()
+        return stateInternal.map { state -> state.feedItems ?: emptyList() }
```

Comment:
Let's also implement `getIsEmpty()`. 
This is needed in order to show an empty view. See `feed_fragment.xml` (look for `empty_text`).